class-two-factor-core.php

```diff
@@ -1433,11 +1631,63 @@ public static function user_two_factor_options( $user ) {
 			$primary_provider_key = null;
 		}
 
-		wp_nonce_field( 'user_two_factor_options', '_nonce_user_two_factor_options', false );
+		// DEBUG START - This section is not intended to be in the final release.
+		if ( $is_current_user ) {
+			$session = WP_Session_Tokens::get_instance( $user->ID )->get( wp_get_session_token() );
+
+			if ( self::is_current_user_session_two_factor() ) {
+				printf(
+					'<div class="notice notice-info inline"><p>%s</p></div>',
+					sprintf(
+						'You are currently logged in with a session from %s. <br>' .
+							'Your 2FA token was last confirmed at %s.<br>' .
+							'Your 2FA method for login was %s (%s after login, %s ago).',
+						'<code>' . date( 'r', $session['login'] ) . '</code>',
+						'<code>' . date( 'r', $session['two-factor-login'] ) . '</code>',
+						'<code>' . esc_html( $session['two-factor-provider'] ) . '</code>',
+						human_time_diff( $session['login'], $session['two-factor-login'] ),
+						human_time_diff( time(), $session['two-factor-login'] ),
+					)
+				);
+			} else {
+				printf(
+					'<div class="notice notice-warning inline"><p>%s</p></div>',
+					sprintf(
+						'You are currently logged in with a session from %s. <br>' .
+							'<strong>You are NOT using a Two Factor session.</strong>',
+						'<code>' . date( 'r', $session['login'] ) . '</code>'
+					)
+				);
+			}
+		}
+		// DEBUG END
```

Comment:
This is for testing only, to make it clearer of what the current login session details are.